superflore/generators/bitbake/gen_packages.py

```diff
@@ -35,51 +33,40 @@
 
 
 def regenerate_installer(
-    overlay, pkg, distro, preserve_existing, tar_dir, md5_cache, sha256_cache
-):
-    make_dir("{0}/recipes-ros-{1}".format(overlay.repo.repo_dir, distro.name))
-    version = get_pkg_version(distro, pkg)
+        overlay, pkg, distro, preserve_existing, tar_dir, md5_cache,
+        sha256_cache, skip_keys):
+    if pkg in skip_keys:
+        warn("package '%s' is on skip-keys, skipping..." % pkg)
+        return None, []
+
+    repo_dir = overlay.repo.repo_dir
+    recipes_dir = '{0}/generated-recipes-{1}'.format(repo_dir, distro.name)
+    make_dir(recipes_dir)
```

Comment:
I feel like there's a lot of unnecessary white space changes being inserted here. Can you reduce this part of the change to only add the `if` check?

Reply:
Sure; I've reduced the unnecessary white spaces while still letting CI pass and for that had to break a few long lines.